compact/ecal/backward_PbWO4.xml

```diff
@@ -14,9 +14,19 @@
     <constant name="EcalEndcapN_structure_ring_max" value="65.00*cm"/>
     <constant name="EcalEndcapN_structure_ring_min" value="EcalEndcapN_structure_ring_max - 0.9*cm"/>
 
+    <comment>Basic hardcoded parameters about rotation and shift</comment>
+    <constant name="EcalEndcapN_Protate" value="15.*degree"/>
```

Comment:
Add more documentation to these variables. It's unclear what Protate or Nrotate are supposed to mean.

Reply:
done